site/content/docs/include/platform.ja.md

```diff
@@ -1,4 +1,20 @@
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
\ No newline at end of file
+<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
+`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）  
+例えば`linux/arm64`や`windows/x86_64`などが指定できます。デフォルトは`linux/x86_64`です。
+
+自動生成された値を上書きして `osfamily` や `architecture` を使うこともできます。  
```

Comment:
I think we don't have to fully localize this sentence. Maybe the following is much simpler and good enough for users? :)
```suggestion
異なる `osfamily` や `architecture` を明示的に指定することもできます。
```

Reply:
I think it's simple and easy to understand! thank you!